packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/examples/SelectedPeopleList.WithContextMenu.Example.tsx

```diff
@@ -87,6 +88,17 @@ export class SelectedPeopleListWithContextMenuExample extends React.Component<
     });
   };
 
+  private _copyToClipboard = (copyString: string): void => {
+    navigator.clipboard.writeText(copyString).then(
+      () => {
+        /* clipboard successfully set */
+      },
+      () => {
+        /* clipboard write failed */
```

Comment:
do we want to throw an error here to know this failed?

Reply:
sure